### ISSUE TITLE
Cannot compile 0.4.5

### ISSUE THREAD

**Issue by jcristee** [citp/BlockSci#70]
Cannot compile latest 0.4.5 no matter what, make always stops at 79%. 0.3 and 0.4.0 were ok. 
Debian 9.4. Cmake 3.10.3.
Log attatched.
[err.txt](https://github.com/citp/BlockSci/files/1839108/err.txt)

**Comment by hkalodner** [comment]
Hi @jcristee, this looks like a recent regression which broke support of gcc-6. I'll get to work on a patch, but in the mean time you could also upgrade to gcc-7 which is working.

**Comment by jcristee** [comment]
Thanks for super fast reply :)
Updated to gcc-7 from debian testing.
Now goes up to 89%.
[err2.txt](https://github.com/citp/BlockSci/files/1839709/err2.txt)

**Comment by hkalodner** [comment]
Happy to help. 

I had never tried compiling on Debian before, but I just played around with it and got it working. To do so I had to add the debian testing repo as described in https://wiki.debian.org/DebianTesting.

To fix the mismatched versions error I ran `sudo apt-get install libssl1.0-dev -t testing` as suggested by https://stackoverflow.com/questions/43151627/installing-g-7-0-1-on-debian-8-7

To fix the RocksDB error I installed the package via `sudo apt install librocksdb-dev -t testing` rather than installing from source.
